Update migration guide regarding InstanceConfigTrait::deleteConfig() (#8124)
cakephp/docs: en/appendices/5-3-migration-guide.rst

Behavior Changes
================

Core
----

> - ``InstanceConfigTrait::deleteConfig()`` has been added. For classes using this

Reviewer: Doesn't matter - but we usually use the phrase `was added` right?

Author: Consistency in grammar is good :)

> - ``InstanceConfigTrait::deleteConfig()`` has been added. For classes using this

Author: ```suggestion
- ``InstanceConfigTrait::deleteConfig()`` was added. For classes using this
```
